components/brave_rewards/core/contribution/contribution_external_wallet.cc

```diff
@@ -147,25 +139,8 @@ void ContributionExternalWallet::OnServerPublisherInfo(
     // the specified |provider| and that the wallet balance is non-zero. We also
     // assume that the user cannot have two connected wallets at the same time.
     // We can then infer that no other external wallet will be able to service
-    // this contribution item, and we can safely put the contribution into the
-    // pending list.
-
+    // this contribution item, and we can safely error out.
     BLOG(1, "Publisher not verified");
-
-    auto save_callback = std::bind(
-        &ContributionExternalWallet::OnSavePendingContribution, this, _1);
-
-    auto contribution = mojom::PendingContribution::New();
```

Comment:
`ledger.mojom.PendingContribution` can now be removed as well. Only reference is in `LedgerDatabaseMigrationTest.Migration_8_PendingContribution`, which doesn't need the type itself. It can just be:
```
TEST_F(LedgerDatabaseMigrationTest, Migration_8_PendingContribution) {
  DatabaseMigration::SetTargetVersionForTesting(8);
  InitializeDatabaseAtVersion(7);
  InitializeLedger();

  EXPECT_EQ(CountTableRows("pending_contribution"), 1);

  sql::Statement info_sql(GetDB()->GetUniqueStatement(R"(
    SELECT amount, added_date, viewing_id, type
    FROM pending_contribution
    WHERE publisher_id = ?
  )"));

  info_sql.BindString(0, "reddit.com");

  ASSERT_TRUE(info_sql.Step());
  EXPECT_EQ(info_sql.ColumnDouble(0), 1.0);
  EXPECT_EQ(static_cast<uint64_t>(info_sql.ColumnInt64(1)),
            UINT64_C(1570614383));
  EXPECT_EQ(info_sql.ColumnString(2), "");
  EXPECT_EQ(info_sql.ColumnInt(3),
            static_cast<int>(mojom::RewardsType::ONE_TIME_TIP));
}
```

Reply:
Same with `ledger.mojom.PendingContributionInfo` and `LedgerDatabaseMigrationTest.Migration_12_ContributionInfo`.